include/alpaka/mem/buf/cpu/Copy.hpp

```diff
@@ -97,8 +97,10 @@ namespace alpaka
                                 m_dstDepth(static_cast<Size>(extent::getDepth(viewDst))),
                                 m_srcDepth(static_cast<Size>(extent::getDepth(viewSrc))),
 #endif
-                                m_dstPitchBytes(static_cast<Size>(mem::view::getPitchBytes<dim::Dim<TViewDst>::value - 1u>(viewDst))),
-                                m_srcPitchBytes(static_cast<Size>(mem::view::getPitchBytes<dim::Dim<TViewSrc>::value - 1u>(viewSrc))),
+                                m_dstPitchBytesX(static_cast<Size>(mem::view::getPitchBytes<dim::Dim<TViewDst>::value - 1u>(viewDst))),
+                                m_srcPitchBytesX(static_cast<Size>(mem::view::getPitchBytes<dim::Dim<TViewSrc>::value - 1u>(viewSrc))),
+                                m_dstPitchBytesY(static_cast<Size>(mem::view::getPitchBytes<dim::Dim<TViewDst>::value - 2u>(viewDst))),                                                                
```

Comment:
Currently you are failing here beause for a 1D Buffer this is asking for the Pitch of the dimension -1.
To fix the current bug, it should be enough to specialize a copy for the 1D case.